install/macos.md

```diff
@@ -76,9 +76,18 @@ Diese startest du mit
 
     mamba activate toolbox
 
-Nach erfolgreicher Installation kannst du die Installationsdatei noch löschen
+Nach erfolgreicher Installation kannst du die Installationsdatei noch löschen. Diese sollte im aktuellen Ordner liegen, um das zu überprüfen, gib einmal
+
+    ls
+
+im Terminal ein. In der ausgegebenen Liste sollte `Mambaforge-Darwin-arm64.sh`, oder ähnlich, auftauchen.
+Wenn das so ist, kannst du sie mit dem folgenden Befehl löschen
+
+    rm Mambaforge-Darwin-arm64.sh
+    
+oder dem entsprechenden Dateinamen.
+Falls du dir nicht sicher bist, kannst du uns für diesen Punkt auch im Workshop ansprechen.
 
-    rm ~/.local/Mambaforge-*.sh
```

Comment:
this was plain wrong, right?
How should this have moved in .local?
Just following the manual one would still be in the home folder I think

Reply:
Do we need all these extra lines, or would it be enough to just fix it? @chrbeckm

Reply:
Those are the lines from the linux/w10 notrs, so yes, the Mambaforge.sh shouldn't be in .local.